miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/display/RealMiniAppView.kt

```diff
@@ -1,7 +1,42 @@
 package com.rakuten.tech.mobile.miniapp.display
 
+import android.annotation.SuppressLint
+import android.content.Context
+import android.view.ViewGroup
+import android.webkit.WebView
+import android.webkit.WebViewClient
+import android.widget.FrameLayout
 import com.rakuten.tech.mobile.miniapp.MiniAppView
 
 internal class RealMiniAppView(
-    val miniAppWebView: MiniAppWebView
-) : MiniAppView
+    val basePath: String
+) : MiniAppView {
+
+    @SuppressLint("SetJavaScriptEnabled")
+    override suspend fun obtainView(context: Context): WebView =
+        WebView(context).apply {
+            setLayoutParams(
+                FrameLayout.LayoutParams(
+                    ViewGroup.LayoutParams.MATCH_PARENT,
+                    ViewGroup.LayoutParams.MATCH_PARENT
+                )
+            )
+            settings.javaScriptEnabled = true
+            setWebViewClient(object : WebViewClient() {
```

Comment:
An anonymous class is okay for now, but probably we'll want to move it out into a separate class when there's more functionality.